mlptrain/log.py

```diff
@@ -10,9 +10,6 @@
 # Try and use colourful logs
 try:
     import coloredlogs
-    FORMAT = '%(asctime)s %(hostname)s %(name)s[%(processName)s] ' \
```

Comment:
What was the reason for removing this again?

Reply:
I've decided to return to using PIDs in log as I've used PIDs in naming PLUMED output files when multiple metadynamics runs execute in parallel.